optimize/client/src/modules/components/UserTypeahead/MultiUserInput.tsx

```diff
@@ -150,28 +150,34 @@ export default function MultiUserInput({
       // disable the internal sorting since we have the data sorted by default
       sortItems={(items) => items}
       initialSelectedItems={selectedUsers}
-      downshiftProps={{
-        onSelect: (item) => {
-          if (!item) {
-            return;
-          }
-
-          const userToRemove = users.find((user) => user.id === item.id);
-
-          if (userToRemove) {
-            onRemove(userToRemove.id);
-          } else {
-            addIdentity(item.id);
-          }
-        },
-      }}
       onChange={({selectedItems}) => {
+        // If no items are selected, clear the selection
         if (selectedItems.length === 0) {
```

Comment:
I am just following the type they are providing. We cannot check every single case where it could possibly break.
I think if they pass something else, then it is a bug from the carbon side.